Unable to build: Undefined symbol _rl_free in mirb.o

```
$ rake
...
Undefined symbols for architecture x86_64:
  "_rl_free", referenced from:
      _main in mirb.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
rake aborted!
```
<details>
<summary>
gem env
</summary>
<pre>
$ gem env
RubyGems Environment:
  - RUBYGEMS VERSION: 3.0.2
  - RUBY VERSION: 2.4.3 (2017-12-14 patchlevel 205) [x86_64-darwin18]
  - INSTALLATION DIRECTORY: /Users/seb/.gem/ruby/2.4.3
  - USER INSTALLATION DIRECTORY: /Users/seb/.gem/ruby/2.4.0
  - RUBY EXECUTABLE: /opt/rubies/2.4.3/bin/ruby
  - GIT EXECUTABLE: /opt/dev/bin/user/git
  - EXECUTABLE DIRECTORY: /Users/seb/.gem/ruby/2.4.3/bin
  - SPEC CACHE DIRECTORY: /Users/seb/.gem/specs
  - SYSTEM CONFIGURATION DIRECTORY: /opt/rubies/2.4.3/etc
  - RUBYGEMS PLATFORMS:
    - ruby
    - x86_64-darwin-18
  - GEM PATHS:
     - /Users/seb/.gem/ruby/2.4.3
     - /opt/rubies/2.4.3/lib/ruby/gems/2.4.0
  - GEM CONFIGURATION:
     - :update_sources => true
     - :verbose => true
     - :backtrace => false
     - :bulk_threshold => 1000
  - REMOTE SOURCES:
     - https://rubygems.org/
  - SHELL PATH:
     - /Users/seb/.gem/ruby/2.4.3/bin
     - /opt/rubies/2.4.3/lib/ruby/gems/2.4.0/bin
     - /opt/rubies/2.4.3/bin
     - /opt/dev/bin/user
     - /Users/seb/.nix-profile/bin
     - /Users/seb/.pyenv/shims
     - /Users/seb/bin
     - /usr/X11/bin
     - /usr/local/bin
     - /usr/sbin
     - /usr/bin
     - /sbin
     - /bin
     - /Users/seb/.composer/vendor/bin
     - /Users/seb/.cargo/bin
</pre>
</details>
<details>
<summary>
Full stacktrace
</summary>
<pre>
$ rake
../../bin/rake
ruby ./minirake
(in /Users/seb/src/github.com/Shopify/ess/ext/enterprise_script_service/mruby)
CC    src/object.c -> build/host/src/object.o
CC    src/kernel.c -> build/host/src/kernel.o
CC    src/print.c -> build/host/src/print.o
CC    src/class.c -> build/host/src/class.o
CC    src/array.c -> build/host/src/array.o
CC    src/etc.c -> build/host/src/etc.o
CC    src/state.c -> build/host/src/state.o
CC    src/init.c -> build/host/src/init.o
CC    src/vm.c -> build/host/src/vm.o
CC    src/string.c -> build/host/src/string.o
CC    src/hash.c -> build/host/src/hash.o
CC    src/codedump.c -> build/host/src/codedump.o
CC    src/debug.c -> build/host/src/debug.o
CC    src/range.c -> build/host/src/range.o
CC    src/fmt_fp.c -> build/host/src/fmt_fp.o
CC    src/dump.c -> build/host/src/dump.o
CC    src/error.c -> build/host/src/error.o
CC    src/backtrace.c -> build/host/src/backtrace.o
CC    src/numeric.c -> build/host/src/numeric.o
CC    src/enum.c -> build/host/src/enum.o
CC    src/symbol.c -> build/host/src/symbol.o
CC    src/proc.c -> build/host/src/proc.o
CC    src/pool.c -> build/host/src/pool.o
CC    src/compar.c -> build/host/src/compar.o
CC    src/version.c -> build/host/src/version.o
CC    src/crc.c -> build/host/src/crc.o
CC    src/load.c -> build/host/src/load.o
CC    src/variable.c -> build/host/src/variable.o
CC    src/gc.c -> build/host/src/gc.o
CC    mrbgems/mruby-bin-mrbc/tools/mrbc/mrbc.c -> build/host/mrbgems/mruby-bin-mrbc/tools/mrbc/mrbc.o
CC    mrbgems/mruby-compiler/core/codegen.c -> build/host/mrbgems/mruby-compiler/core/codegen.o
YACC  mrbgems/mruby-compiler/core/parse.y -> build/host/mrbgems/mruby-compiler/core/y.tab.c
CC    build/host/mrbgems/mruby-compiler/core/y.tab.c -> build/host/mrbgems/mruby-compiler/core/y.tab.o
AR    build/host/lib/libmruby_core.a
ar: creating archive /Users/seb/src/github.com/Shopify/ess/ext/enterprise_script_service/mruby/build/host/lib/libmruby_core.a
LD    build/host/bin/mrbc
GEN   *.rb -> build/host/mrblib/mrblib.c
      MRBC mrblib/00class.rb
      MRBC mrblib/10error.rb
      MRBC mrblib/array.rb
      MRBC mrblib/compar.rb
      MRBC mrblib/enum.rb
      MRBC mrblib/float.rb
      MRBC mrblib/hash.rb
      MRBC mrblib/kernel.rb
      MRBC mrblib/numeric.rb
      MRBC mrblib/range.rb
      MRBC mrblib/string.rb
CC    build/host/mrblib/mrblib.c -> build/host/mrblib/mrblib.o
CC    ../mruby-mpdecimal/src/fnt.c -> build/host/mrbgems/mruby-mpdecimal/src/fnt.o
CC    ../mruby-mpdecimal/src/convolute.c -> build/host/mrbgems/mruby-mpdecimal/src/convolute.o
CC    ../mruby-mpdecimal/src/mpsignal.c -> build/host/mrbgems/mruby-mpdecimal/src/mpsignal.o
CC    ../mruby-mpdecimal/src/difradix2.c -> build/host/mrbgems/mruby-mpdecimal/src/difradix2.o
CC    ../mruby-mpdecimal/src/constants.c -> build/host/mrbgems/mruby-mpdecimal/src/constants.o
CC    ../mruby-mpdecimal/src/memory.c -> build/host/mrbgems/mruby-mpdecimal/src/memory.o
CC    ../mruby-mpdecimal/src/crt.c -> build/host/mrbgems/mruby-mpdecimal/src/crt.o
CC    ../mruby-mpdecimal/src/mpdecimal.c -> build/host/mrbgems/mruby-mpdecimal/src/mpdecimal.o
CC    ../mruby-mpdecimal/src/io.c -> build/host/mrbgems/mruby-mpdecimal/src/io.o
CC    ../mruby-mpdecimal/src/ext.c -> build/host/mrbgems/mruby-mpdecimal/src/ext.o
CC    ../mruby-mpdecimal/src/transpose.c -> build/host/mrbgems/mruby-mpdecimal/src/transpose.o
CC    ../mruby-mpdecimal/src/basearith.c -> build/host/mrbgems/mruby-mpdecimal/src/basearith.o
CC    ../mruby-mpdecimal/src/numbertheory.c -> build/host/mrbgems/mruby-mpdecimal/src/numbertheory.o
CC    ../mruby-mpdecimal/src/sixstep.c -> build/host/mrbgems/mruby-mpdecimal/src/sixstep.o
CC    ../mruby-mpdecimal/src/context.c -> build/host/mrbgems/mruby-mpdecimal/src/context.o
CC    ../mruby-mpdecimal/src/fourstep.c -> build/host/mrbgems/mruby-mpdecimal/src/fourstep.o
      MRBC ../mruby-mpdecimal/mrblib/mpdecimal.rb
CC    build/host/mrbgems/mruby-mpdecimal/gem_init.c -> build/host/mrbgems/mruby-mpdecimal/gem_init.o
CC    mrbgems/mruby-math/src/math.c -> build/host/mrbgems/mruby-math/src/math.o
CC    build/host/mrbgems/mruby-math/gem_init.c -> build/host/mrbgems/mruby-math/gem_init.o
CC    mrbgems/mruby-struct/src/struct.c -> build/host/mrbgems/mruby-struct/src/struct.o
      MRBC mrbgems/mruby-struct/mrblib/struct.rb
CC    build/host/mrbgems/mruby-struct/gem_init.c -> build/host/mrbgems/mruby-struct/gem_init.o
      MRBC mrbgems/mruby-enum-ext/mrblib/enum.rb
CC    build/host/mrbgems/mruby-enum-ext/gem_init.c -> build/host/mrbgems/mruby-enum-ext/gem_init.o
CC    mrbgems/mruby-string-ext/src/string.c -> build/host/mrbgems/mruby-string-ext/src/string.o
      MRBC mrbgems/mruby-string-ext/mrblib/string.rb
CC    build/host/mrbgems/mruby-string-ext/gem_init.c -> build/host/mrbgems/mruby-string-ext/gem_init.o
CC    mrbgems/mruby-numeric-ext/src/numeric_ext.c -> build/host/mrbgems/mruby-numeric-ext/src/numeric_ext.o
      MRBC mrbgems/mruby-numeric-ext/mrblib/numeric_ext.rb
CC    build/host/mrbgems/mruby-numeric-ext/gem_init.c -> build/host/mrbgems/mruby-numeric-ext/gem_init.o
CC    mrbgems/mruby-array-ext/src/array.c -> build/host/mrbgems/mruby-array-ext/src/array.o
      MRBC mrbgems/mruby-array-ext/mrblib/array.rb
CC    build/host/mrbgems/mruby-array-ext/gem_init.c -> build/host/mrbgems/mruby-array-ext/gem_init.o
CC    mrbgems/mruby-hash-ext/src/hash-ext.c -> build/host/mrbgems/mruby-hash-ext/src/hash-ext.o
      MRBC mrbgems/mruby-hash-ext/mrblib/hash.rb
CC    build/host/mrbgems/mruby-hash-ext/gem_init.c -> build/host/mrbgems/mruby-hash-ext/gem_init.o
CC    mrbgems/mruby-range-ext/src/range.c -> build/host/mrbgems/mruby-range-ext/src/range.o
      MRBC mrbgems/mruby-range-ext/mrblib/range.rb
CC    build/host/mrbgems/mruby-range-ext/gem_init.c -> build/host/mrbgems/mruby-range-ext/gem_init.o
CC    mrbgems/mruby-proc-ext/src/proc.c -> build/host/mrbgems/mruby-proc-ext/src/proc.o
      MRBC mrbgems/mruby-proc-ext/mrblib/proc.rb
CC    build/host/mrbgems/mruby-proc-ext/gem_init.c -> build/host/mrbgems/mruby-proc-ext/gem_init.o
CC    mrbgems/mruby-symbol-ext/src/symbol.c -> build/host/mrbgems/mruby-symbol-ext/src/symbol.o
      MRBC mrbgems/mruby-symbol-ext/mrblib/symbol.rb
CC    build/host/mrbgems/mruby-symbol-ext/gem_init.c -> build/host/mrbgems/mruby-symbol-ext/gem_init.o
CC    mrbgems/mruby-object-ext/src/object.c -> build/host/mrbgems/mruby-object-ext/src/object.o
      MRBC mrbgems/mruby-object-ext/mrblib/object.rb
CC    build/host/mrbgems/mruby-object-ext/gem_init.c -> build/host/mrbgems/mruby-object-ext/gem_init.o
      MRBC mrbgems/mruby-toplevel-ext/mrblib/toplevel.rb
CC    build/host/mrbgems/mruby-toplevel-ext/gem_init.c -> build/host/mrbgems/mruby-toplevel-ext/gem_init.o
CC    mrbgems/mruby-kernel-ext/src/kernel.c -> build/host/mrbgems/mruby-kernel-ext/src/kernel.o
      MRBC mrbgems/mruby-kernel-ext/mrblib/kernel.rb
CC    build/host/mrbgems/mruby-kernel-ext/gem_init.c -> build/host/mrbgems/mruby-kernel-ext/gem_init.o
CC    mrbgems/mruby-error/src/exception.c -> build/host/mrbgems/mruby-error/src/exception.o
CC    build/host/mrbgems/mruby-error/gem_init.c -> build/host/mrbgems/mruby-error/gem_init.o
CC    build/host/mrbgems/gem_init.c -> build/host/mrbgems/gem_init.o
AR    build/host/lib/libmruby.a
ar: creating archive /Users/seb/src/github.com/Shopify/ess/ext/enterprise_script_service/mruby/build/host/lib/libmruby.a
CC    src/object.c -> build/sandbox/src/object.o
CC    src/kernel.c -> build/sandbox/src/kernel.o
CC    src/print.c -> build/sandbox/src/print.o
CC    src/class.c -> build/sandbox/src/class.o
CC    src/array.c -> build/sandbox/src/array.o
CC    src/etc.c -> build/sandbox/src/etc.o
CC    src/state.c -> build/sandbox/src/state.o
CC    src/init.c -> build/sandbox/src/init.o
CC    src/vm.c -> build/sandbox/src/vm.o
CC    src/string.c -> build/sandbox/src/string.o
CC    src/hash.c -> build/sandbox/src/hash.o
CC    src/codedump.c -> build/sandbox/src/codedump.o
CC    src/debug.c -> build/sandbox/src/debug.o
CC    src/range.c -> build/sandbox/src/range.o
CC    src/fmt_fp.c -> build/sandbox/src/fmt_fp.o
CC    src/dump.c -> build/sandbox/src/dump.o
CC    src/error.c -> build/sandbox/src/error.o
CC    src/backtrace.c -> build/sandbox/src/backtrace.o
CC    src/numeric.c -> build/sandbox/src/numeric.o
CC    src/enum.c -> build/sandbox/src/enum.o
CC    src/symbol.c -> build/sandbox/src/symbol.o
CC    src/proc.c -> build/sandbox/src/proc.o
CC    src/pool.c -> build/sandbox/src/pool.o
CC    src/compar.c -> build/sandbox/src/compar.o
CC    src/version.c -> build/sandbox/src/version.o
CC    src/crc.c -> build/sandbox/src/crc.o
CC    src/load.c -> build/sandbox/src/load.o
CC    src/variable.c -> build/sandbox/src/variable.o
CC    src/gc.c -> build/sandbox/src/gc.o
GEN   *.rb -> build/sandbox/mrblib/mrblib.c
      MRBC mrblib/00class.rb
      MRBC mrblib/10error.rb
      MRBC mrblib/array.rb
      MRBC mrblib/compar.rb
      MRBC mrblib/enum.rb
      MRBC mrblib/float.rb
      MRBC mrblib/hash.rb
      MRBC mrblib/kernel.rb
      MRBC mrblib/numeric.rb
      MRBC mrblib/range.rb
      MRBC mrblib/string.rb
CC    build/sandbox/mrblib/mrblib.c -> build/sandbox/mrblib/mrblib.o
CC    ../mruby-mpdecimal/src/fnt.c -> build/sandbox/mrbgems/mruby-mpdecimal/src/fnt.o
CC    ../mruby-mpdecimal/src/convolute.c -> build/sandbox/mrbgems/mruby-mpdecimal/src/convolute.o
CC    ../mruby-mpdecimal/src/mpsignal.c -> build/sandbox/mrbgems/mruby-mpdecimal/src/mpsignal.o
CC    ../mruby-mpdecimal/src/difradix2.c -> build/sandbox/mrbgems/mruby-mpdecimal/src/difradix2.o
CC    ../mruby-mpdecimal/src/constants.c -> build/sandbox/mrbgems/mruby-mpdecimal/src/constants.o
CC    ../mruby-mpdecimal/src/memory.c -> build/sandbox/mrbgems/mruby-mpdecimal/src/memory.o
CC    ../mruby-mpdecimal/src/crt.c -> build/sandbox/mrbgems/mruby-mpdecimal/src/crt.o
CC    ../mruby-mpdecimal/src/mpdecimal.c -> build/sandbox/mrbgems/mruby-mpdecimal/src/mpdecimal.o
CC    ../mruby-mpdecimal/src/io.c -> build/sandbox/mrbgems/mruby-mpdecimal/src/io.o
CC    ../mruby-mpdecimal/src/ext.c -> build/sandbox/mrbgems/mruby-mpdecimal/src/ext.o
CC    ../mruby-mpdecimal/src/transpose.c -> build/sandbox/mrbgems/mruby-mpdecimal/src/transpose.o
CC    ../mruby-mpdecimal/src/basearith.c -> build/sandbox/mrbgems/mruby-mpdecimal/src/basearith.o
CC    ../mruby-mpdecimal/src/numbertheory.c -> build/sandbox/mrbgems/mruby-mpdecimal/src/numbertheory.o
CC    ../mruby-mpdecimal/src/sixstep.c -> build/sandbox/mrbgems/mruby-mpdecimal/src/sixstep.o
CC    ../mruby-mpdecimal/src/context.c -> build/sandbox/mrbgems/mruby-mpdecimal/src/context.o
CC    ../mruby-mpdecimal/src/fourstep.c -> build/sandbox/mrbgems/mruby-mpdecimal/src/fourstep.o
      MRBC ../mruby-mpdecimal/mrblib/mpdecimal.rb
CC    build/sandbox/mrbgems/mruby-mpdecimal/gem_init.c -> build/sandbox/mrbgems/mruby-mpdecimal/gem_init.o
CC    mrbgems/mruby-math/src/math.c -> build/sandbox/mrbgems/mruby-math/src/math.o
CC    build/sandbox/mrbgems/mruby-math/gem_init.c -> build/sandbox/mrbgems/mruby-math/gem_init.o
CC    mrbgems/mruby-struct/src/struct.c -> build/sandbox/mrbgems/mruby-struct/src/struct.o
      MRBC mrbgems/mruby-struct/mrblib/struct.rb
CC    build/sandbox/mrbgems/mruby-struct/gem_init.c -> build/sandbox/mrbgems/mruby-struct/gem_init.o
      MRBC mrbgems/mruby-enum-ext/mrblib/enum.rb
CC    build/sandbox/mrbgems/mruby-enum-ext/gem_init.c -> build/sandbox/mrbgems/mruby-enum-ext/gem_init.o
CC    mrbgems/mruby-string-ext/src/string.c -> build/sandbox/mrbgems/mruby-string-ext/src/string.o
      MRBC mrbgems/mruby-string-ext/mrblib/string.rb
CC    build/sandbox/mrbgems/mruby-string-ext/gem_init.c -> build/sandbox/mrbgems/mruby-string-ext/gem_init.o
CC    mrbgems/mruby-numeric-ext/src/numeric_ext.c -> build/sandbox/mrbgems/mruby-numeric-ext/src/numeric_ext.o
      MRBC mrbgems/mruby-numeric-ext/mrblib/numeric_ext.rb
CC    build/sandbox/mrbgems/mruby-numeric-ext/gem_init.c -> build/sandbox/mrbgems/mruby-numeric-ext/gem_init.o
CC    mrbgems/mruby-array-ext/src/array.c -> build/sandbox/mrbgems/mruby-array-ext/src/array.o
      MRBC mrbgems/mruby-array-ext/mrblib/array.rb
CC    build/sandbox/mrbgems/mruby-array-ext/gem_init.c -> build/sandbox/mrbgems/mruby-array-ext/gem_init.o
CC    mrbgems/mruby-hash-ext/src/hash-ext.c -> build/sandbox/mrbgems/mruby-hash-ext/src/hash-ext.o
      MRBC mrbgems/mruby-hash-ext/mrblib/hash.rb
CC    build/sandbox/mrbgems/mruby-hash-ext/gem_init.c -> build/sandbox/mrbgems/mruby-hash-ext/gem_init.o
CC    mrbgems/mruby-range-ext/src/range.c -> build/sandbox/mrbgems/mruby-range-ext/src/range.o
      MRBC mrbgems/mruby-range-ext/mrblib/range.rb
CC    build/sandbox/mrbgems/mruby-range-ext/gem_init.c -> build/sandbox/mrbgems/mruby-range-ext/gem_init.o
CC    mrbgems/mruby-proc-ext/src/proc.c -> build/sandbox/mrbgems/mruby-proc-ext/src/proc.o
      MRBC mrbgems/mruby-proc-ext/mrblib/proc.rb
CC    build/sandbox/mrbgems/mruby-proc-ext/gem_init.c -> build/sandbox/mrbgems/mruby-proc-ext/gem_init.o
CC    mrbgems/mruby-symbol-ext/src/symbol.c -> build/sandbox/mrbgems/mruby-symbol-ext/src/symbol.o
      MRBC mrbgems/mruby-symbol-ext/mrblib/symbol.rb
CC    build/sandbox/mrbgems/mruby-symbol-ext/gem_init.c -> build/sandbox/mrbgems/mruby-symbol-ext/gem_init.o
CC    mrbgems/mruby-object-ext/src/object.c -> build/sandbox/mrbgems/mruby-object-ext/src/object.o
      MRBC mrbgems/mruby-object-ext/mrblib/object.rb
CC    build/sandbox/mrbgems/mruby-object-ext/gem_init.c -> build/sandbox/mrbgems/mruby-object-ext/gem_init.o
      MRBC mrbgems/mruby-toplevel-ext/mrblib/toplevel.rb
CC    build/sandbox/mrbgems/mruby-toplevel-ext/gem_init.c -> build/sandbox/mrbgems/mruby-toplevel-ext/gem_init.o
CC    mrbgems/mruby-kernel-ext/src/kernel.c -> build/sandbox/mrbgems/mruby-kernel-ext/src/kernel.o
      MRBC mrbgems/mruby-kernel-ext/mrblib/kernel.rb
CC    build/sandbox/mrbgems/mruby-kernel-ext/gem_init.c -> build/sandbox/mrbgems/mruby-kernel-ext/gem_init.o
CC    mrbgems/mruby-compiler/core/codegen.c -> build/sandbox/mrbgems/mruby-compiler/core/codegen.o
YACC  mrbgems/mruby-compiler/core/parse.y -> build/sandbox/mrbgems/mruby-compiler/core/y.tab.c
CC    build/sandbox/mrbgems/mruby-compiler/core/y.tab.c -> build/sandbox/mrbgems/mruby-compiler/core/y.tab.o
CC    build/sandbox/mrbgems/gem_init.c -> build/sandbox/mrbgems/gem_init.o
AR    build/sandbox/lib/libmruby.a
ar: creating archive /Users/seb/src/github.com/Shopify/ess/ext/enterprise_script_service/mruby/build/sandbox/lib/libmruby.a
CC    mrbgems/mruby-bin-mruby/tools/mruby/mruby.c -> build/host/mrbgems/mruby-bin-mruby/tools/mruby/mruby.o
LD    build/host/bin/mruby
CC    mrbgems/mruby-bin-mirb/tools/mirb/mirb.c -> build/host/mrbgems/mruby-bin-mirb/tools/mirb/mirb.o
LD    build/host/bin/mirb
Undefined symbols for architecture x86_64:
  "_rl_free", referenced from:
      _main in mirb.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
Undefined symbols for architecture x86_64:
  "_rl_free", referenced from:
      _main in mirb.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
rake aborted!
Command Failed: [gcc  -o "/Users/seb/src/github.com/Shopify/ess/ext/enterprise_script_service/mruby/build/host/bin/mirb" "/Users/seb/src/github.com/Shopify/ess/ext/enterprise_script_service/mruby/build/host/mrbgems/mruby-bin-mirb/tools/mirb/mirb.o" "/Users/seb/src/github.com/Shopify/ess/ext/enterprise_script_service/mruby/build/host/lib/libmruby.a"  -lm -lreadline -lncurses ]
rakefile:74:in `block (4 levels) in <top (required)>'
rake aborted!
Command failed with status (1): [ruby ./minirake...]
/Users/seb/src/github.com/Shopify/ess/ext/enterprise_script_service/Rakefile:130:in `block (3 levels) in <top (required)>'
/Users/seb/src/github.com/Shopify/ess/ext/enterprise_script_service/Rakefile:121:in `block in within_mruby'
/Users/seb/src/github.com/Shopify/ess/ext/enterprise_script_service/Rakefile:117:in `chdir'
/Users/seb/src/github.com/Shopify/ess/ext/enterprise_script_service/Rakefile:117:in `within_mruby'
/Users/seb/src/github.com/Shopify/ess/ext/enterprise_script_service/Rakefile:129:in `block (2 levels) in <top (required)>'
/Users/seb/.gem/ruby/2.4.3/gems/rake-12.3.2/exe/rake:27:in `<top (required)>'
Tasks: TOP => default => /Users/seb/src/github.com/Shopify/ess/bin/enterprise_script_service => /Users/seb/src/github.com/Shopify/ess/ext/enterprise_script_service/mruby/build/sandbox/lib/libmruby.a => mruby:compile
(See full trace by running task with --trace)
rake aborted!
Command failed with status (1): [../../bin/rake...]
/Users/seb/src/github.com/Shopify/ess/Rakefile:12:in `block (2 levels) in <top (required)>'
/Users/seb/src/github.com/Shopify/ess/Rakefile:11:in `chdir'
/Users/seb/src/github.com/Shopify/ess/Rakefile:11:in `block in <top (required)>'
/Users/seb/.gem/ruby/2.4.3/gems/rake-12.3.2/exe/rake:27:in `<top (required)>'
Tasks: TOP => default => spec => compile
(See full trace by running task with --trace)
</pre>
</details>

--- a/ext/enterprise_script_service/flags.rb
+++ b/ext/enterprise_script_service/flags.rb
@@ -16,6 +16,11 @@ module Flags
       end
     end
 
+    def library_paths
+      # Necessary because of https://github.com/mruby/mruby/issues/4537
+      %w(/usr/local/lib /usr/lib)
+    end
+
     def io_safe_defines
       %w(
         _GNU_SOURCE

--- a/ext/enterprise_script_service/mruby_config.rb
+++ b/ext/enterprise_script_service/mruby_config.rb
@@ -2,6 +2,8 @@ require_relative("./flags")
 
 mruby_engine_gembox_path = Pathname.new(__FILE__).dirname.join("mruby_engine")
 
+# https://github.com/mruby/mruby/blob/master/doc/guides/compile.md
+
 MRuby::Build.new do |conf|
   toolchain(:gcc)
 
@@ -18,6 +20,10 @@ MRuby::Build.new do |conf|
     cc.flags += Flags.cflags
     cc.defines += Flags.io_safe_defines
   end
+
+  conf.linker do |linker|
+    linker.library_paths += Flags.library_paths
+  end
 end
 
 MRuby::CrossBuild.new("sandbox") do |conf|
@@ -33,5 +39,9 @@ MRuby::CrossBuild.new("sandbox") do |conf|
     cc.flags += %w(-fPIC)
     cc.flags += Flags.cflags
     cc.defines += Flags.defines
+  end
+
+  conf.linker do |linker|
+    linker.library_paths += Flags.library_paths
   end
 end

